templates/coderd.yaml

```diff
@@ -229,6 +242,13 @@ metadata:
   name: {{ include "coder.serviceName" . }}
   {{- end }}
   namespace: {{ .Release.Namespace | quote }}
+  labels:
+    app.kubernetes.io/name: {{ .Chart.Name }}
```

Comment:
these labels make it possible to query for all objects corresponding to a given version of Coder, and let you search (e.g. list all coder 1.25 installs in a cluster)